packages/core/src/baseclient.ts

```diff
@@ -152,6 +152,11 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
    * @inheritDoc
    */
   public captureSession(session: Session): void {
+    if (!this._isEnabled()) {
+      logger.warn('SDK not enabled, will not send session.');
```

Comment:
```suggestion
      logger.warn('SDK not enabled, cannot capture session.');
```

Just since this is in the `captureSession` method and not the `sendSession` method.

Reply:
Then remember to update `will not send event` too :)